bundles/org.openhab.binding.emby/src/main/resources/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="emby" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Emby Binding</name>
+	<description>This is the binding for Emby.</description>
```

Comment:
Could you be a bit more verbose, so that somebody who hasn't heard about it gets a clue what this is about?